### PR TITLE
Add ilastik collection

### PR DESCRIPTION
**Pull request by constantinpape**

### CODE REVIEW

**Comment by oeway** [comment]
If you want to link to the Ilastik app, you should also move the ilastik app to your repo:
```yaml
application:
  - id: Ilastik
    source: src/Ilastik-app.imjoy.html
```
As mentioned in the other issue, you may want to improve the content in `Ilastik-app.imjoy.html`.

File: src/manifest.bioimage.io.yaml [hunk 1]
```diff
@@ -55,24 +57,6 @@ model:
...
-      - Ilastik
-    download_url: https://github.com/wolny/pytorch-3dunet/releases/download/1.2.6/UNet3DArabidopsisOvules.model.zip
-
-  - id: UNet3DPlatyCellProbs
-    source: https://raw.githubusercontent.com/platybrowser/platybrowser/3711f1c26e5db8c38c3faff4cccb3110560e3c67/segmentation/cells/UNet3DPlatyCellProbs.model/UNet3DPlatyCellProbs.model.yaml
-    links:
-      - Ilastik
-    download_url: https://github.com/platybrowser/platybrowser/releases/download/1.0.0/UNet3DPlatyCellProbs.model.zip
-
   - id: UNetDA
```
Comment (oeway) [inline src/manifest.bioimage.io.yaml]: Will you also take `UNetDA` and `2sUNetDA`?

**Comment by constantinpape** [comment]
> As mentioned in the other issue, you may want to improve the content in `Ilastik-app.imjoy.html`

Is this the online preview tool for ilastik models?
Then yeah, makes sense to take it.

File: src/manifest.bioimage.io.yaml (hunk 1 above)
Comment (constantinpape) [reply src/manifest.bioimage.io.yaml]: Not really sure what these models are.
Should we add them to the ilastik models @FynnBe ?

**Comment by oeway** [comment]
> > As mentioned in the other issue, you may want to improve the content in `Ilastik-app.imjoy.html`
> 
> Is this the online preview tool for ilastik models?
> Then yeah, makes sense to take it.

Yes. I renamed it into `Ilastik`, feel free to make changes. I think you can treat it as a landing page for Ilastik, add more description about Ilastik, add download links, even embed a youtube videos into it. You can also optionally enable the preview button.

**Comment by constantinpape** [comment]
@oeway I think this is good to go now; let me know if there are any more changes necessary.

**Review by oeway — APPROVED** [review]
looks good

**Pull request merged**